round4/team-13.md

```diff
@@ -0,0 +1,57 @@
+# Chrome Extension Idea: TypeMaster (tentative)
+
+## Authors
+
+Tiffany Lian, Alexander Dempsey, James Huang, Sanya Shetty
+
+## Problem Statement
+
+Many application portals require users to meet word count limits which can be both challenging and demotivating. Without any built-in word count features on the portal users would have to resort to external tools like pasting into Google Docs which is inefficient. Moreover, integrating AI-powered semantic analysis and real-time analytics into the user interface could streamline the process, providing insightful feedback and enhancing productivity and user experience.
+
+## Target Audience
+
+- The target audience for WordCounter includes:
+
+- Students working on assignments with word requirements set by their instructors.
+
+- Editors and Proofreaders who want to estimate the time required for a task.
+
+- Applicants applying to jobs, clubs, or colleges who have prompts with word requirements.
+
+
+## Description
+
+TypeMaster is a portable writing assistant that provides word counter, writing analytics, and helps users visualize their writing goals on any website, 
+
+## Selling Points
+
+1. Allows uninterrupted workflow, accessible from within the tab.
+2. Achieve writing milestones effortlessly by setting customizable word count goals and receiving visual prompts as you progress.
+3. Persistent settings across browsing sessions and websites
+4. Maximize productivity with quick and easy access.
+5. Accurate word counts and analytics displayed in a compact, unobtrusive manner.
+6. OpenAI API integration which provides you feedback on your professional applications.
```

Comment:
A bit confused at what this might be, but this sounds promising! And using OpenAI's API is always good practice for the future too

Reply:
The idea is for ChatGPT to provide feedback on style and other tips for resume/professional writing!